tyr/clusters/autoscaling.py

```diff
@@ -90,7 +109,14 @@ def create_autoscaling_group(self):
             self.log.info("Creating new autoscaling group: {g}"
                           .format(g=self.autoscaling_group))
 
+	    # Convert our tags list into something that AWS can understand:
+	    aws_tags = list()
```

Comment:
An empty list would just be `[]`, but you could probably replace the whole loop with

```
aws_tags = [Tag(resource_id=self.autoscaling_group, key=tag['name'],
                value=tag['value'], propagate_at_launch=True) for tag in self.tags]
```